__tests__/app.test.js

```diff
@@ -138,13 +138,22 @@ describe("200: /api/articles", () => {
 
       });
   });
-  test("should return error with msg Not Found for request not in database", () => {
+  test("should return 200 with empty array for request with no comments", () => {
     return request(app)
-      .get("/api/articles/99999999/comments")
-      .expect(404)
+      .get("/api/articles/37/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toEqual([]);
+      });
```

Comment:
This test is great and what I asked for, but you replaced an older test that we still need! 

 test("should return error with msg Not Found for request not in database", () => {
    return request(app)
      .get("/api/articles/99999999/comments")
      .expect(404)
      } 
      
You need to implement both of these at the same time, differentiating between existing articles with no comments, and non existent articles, _both_ of these situations return empty rows [] so you need to create a function that checks the article exists first :)